ephios/plugins/files/templates/files/document_list.html

```diff
@@ -0,0 +1,33 @@
+{% extends "core/settings/settings_base.html" %}
+{% load i18n %}
+
+{% block settings_content %}
+    <a class="btn btn-secondary" href="{% url "files:settings_document_create" %}"><span
+        class="fa fa-plus"></span> {% translate "Add file" %}</a>
+    <table class="table table-striped display mt-2">
+        <thead>
+            <tr>
+                <th>{% translate "Title" %}</th>
+                <th>{% translate "Action" %}</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for document in document_list %}
+                <tr>
+                    <td class="break-word">{{ document.title }}</td>
```

Comment:
I think it would be great to show uploader name and last modified date in the list, e.g. to look through and see old/outdated documents.